libs/ledger-live-common/src/families/icon/api/node.ts

```diff
@@ -0,0 +1,107 @@
+import { BigNumber } from "bignumber.js";
+import IconService from "icon-sdk-js";
+import type { IcxTransaction } from "icon-sdk-js";
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import { getEnv } from "@ledgerhq/live-env";
+import { isTestnet, roundedLoopAmount } from "../logic";
+import { GOVERNANCE_SCORE_ADDRESS, IISS_SCORE_ADDRESS } from "../constants";
+import { IconAccount } from "../types";
+import { SignedOperation } from "@ledgerhq/types-live";
+
+const { HttpProvider } = IconService;
+const { IconBuilder } = IconService;
+
+/**
+ * Returns Testnet RPC URL if the current currency is testnet
+ * @param {currency} currency
+ */
+export function getRpcUrl(currency: CryptoCurrency): string {
+  let rpcUrl = getEnv("ICON_NODE_ENDPOINT");
+  if (isTestnet(currency)) {
+    rpcUrl = getEnv("ICON_TESTNET_NODE_ENDPOINT");
+  }
+  return rpcUrl;
+}
+
+/**
+ * Broadcast blob to blockchain
+ */
+export const broadcastTransaction = async (
+  signedOperation: SignedOperation,
+  currency: CryptoCurrency,
+) => {
+  const { hash } = await submit(signedOperation, currency);
+  // Transaction hash is likely to be returned
```

Comment:
Is it possible to not get a hash here ?

Reply:
Hi @hedi-edelbloute, I'm a new member of the Techiast team. I will continue contributing to this PR. I confirmed a `hash` return when sent successfully. I removed this line "// Transaction hash is likely to be returned"